### PR TITLE
INFRA-310: Add drugs variant loader

### PR DESCRIPTION
**Pull request by chinzou**
https://mekomsolutions.atlassian.net/browse/INFRA-310

### CODE REVIEW

File: readme/drug.md [hunk 1]
```diff
@@ -2,36 +2,14 @@
...
-    "product_variant_ids/uuid": "Uuid",
-    "name": "Name",
-    "lst_price": "odoo_price",
-    "product_variant_ids/categ_id/id": "odoo_category",
-    "type":"odoo_type",
-    "id": "odoo_id",
-    "uom_id/id": "odoo_uom",
-    "standard_price": "odoo_cost"
+    "product_variant_ids/id": "odoo_id",
+    "id": "odoo_id"
```
Comment (rbuisson) [inline readme/drug.md:12]: Should we make the change and support 2 non-identical fields?
Maybe `odoo_variant_id`

File: odoo_initializer/models/drug_loader.py [hunk 2]
```diff
@@ -4,15 +4,8 @@
 class DrugLoader(BaseLoader):
     model_name = "product.template"
     field_mapping = {
-        "lst_price": "odoo_price",
-        "product_variant_ids/categ_id/id": "odoo_category",
-        "type":"odoo_type",
-        "name": "Name",
-        "product_variant_ids/uuid": "Uuid",
+        "product_variant_ids/id": "odoo_id",
```
Comment (rbuisson) [inline odoo_initializer/models/drug_loader.py:7]: Should we make the change and support 2 non-identical fields?
Maybe `odoo_variant_id`

**Review by rbuisson — CHANGES_REQUESTED** [review]

File: odoo_initializer/tests/resources/config/initz_config_file.json [hunk 3]
```diff
@@ -0,0 +1,18 @@
+{
```
Comment (rbuisson) [inline odoo_initializer/tests/resources/config/initz_config_file.json:1]: Not sure what this file is about.
Isn't there already a test resource for this? [Here](https://github.com/mekomsolutions/odoo-initializer/tree/master/odoo_initializer/tests/resources/config)

Additionnaly the name of the file is not correct. Should be `iniz` not `initz`

**Review by rbuisson — CHANGES_REQUESTED** [review]

File: odoo_initializer/models/drug_variant_loader.py [hunk 4]
```diff
@@ -0,0 +1,18 @@
...
+
+class DrugVariantLoader(BaseLoader):
+    model_name = "product.product"
+    field_mapping = {
+        "lst_price": "odoo_price",
+        "categ_id/id": "odoo_category",
+        "type":"odoo_type",
+        "name": "Name",
+        "uuid": "Uuid",
+        "id": "odoo_id",
```
Comment (rbuisson) [inline odoo_initializer/models/drug_variant_loader.py:12]: We should add the `odoo_variant_id` that maps to the `product_variant_ids/id`.

**Review by rbuisson — CHANGES_REQUESTED** [review]

File: odoo_initializer/tests/resources/config/initz_config_file.json (hunk 3 above)
Comment (chinzou) [reply odoo_initializer/tests/resources/config/initz_config_file.json:1]: This is file is not useful, will delete it

File: readme/drug.md (hunk 1 above)
Comment (rbuisson) [reply readme/drug.md]: won't be used anymore as we will only load the drugs via the variants model.

File: odoo_initializer/models/drug_loader.py [hunk 5]
```diff
@@ -2,13 +2,13 @@
...
-        "product_variant_ids/categ_id/id": "odoo_category",
-        "type":"odoo_type",
+        "categ_id/id": "odoo_category",
+        "type": "odoo_type",
         "name": "Name",
-        "product_variant_ids/uuid": "Uuid",
+        "uuid": "Uuid",
         "id": "odoo_id",
         "uom_id/id": "odoo_uom",
         "standard_price": "odoo_cost",
```
Comment (rbuisson) [inline odoo_initializer/models/drug_loader.py:14]: Should we add:
```
field_rules = {
        "lst_price": "NO_UPDATE",
        "standard_price": "NO_UPDATE"
    }
```
To make sure we don't override the prices unintentionally?

**Review by rbuisson — CHANGES_REQUESTED** [review]

**Pull request merged**